IABot/tests/acceptance/test_cases/acceptance-result.txt

```diff
@@ -0,0 +1,17 @@
+Some text.
+<ref name="raw-live-url">https://en.wikipedia.org/wiki/Main_Page</ref>
+<ref name="raw-dead-url">{{cite web|url=http://www.statistics.sk/mosmis/eng/run.html |title=Archived copy |accessdate=2007-05-13 |deadurl=yes |archiveurl=https://web.archive.org/web/20070513023228/http://www.statistics.sk/mosmis/eng/run.html |archivedate=2007-05-28 |df= }}</ref>
+<ref name="raw-dead-url-w-text">Doe, Jane (1977). http://www.statistics.sk/mosmis/eng/run.html.{{dead link}} More text.</ref>
+<ref name="raw-dead-link">[http://www.statistics.sk/mosmis/eng/run.html "Statistics"] {{wayback|url=http://www.statistics.sk/mosmis/eng/run.html |date=20070528023228 }}</ref>
+<ref name="raw-dead-link-w-text">Doe, Jane (1977). [http://www.statistics.sk/mosmis/eng/run.html "Statistics"] {{wayback|url=http://www.statistics.sk/mosmis/eng/run.html |date=20070528023228 }}. More text.</ref>
+<ref name="raw-dead-link-w-archive">[http://www.statistics.sk/mosmis/eng/run.html "Statistics"]. ([https://web.archive.org/web/20070513023228/http://www.statistics.sk/mosmis/eng/run.html Archive])</ref>
+<ref name="raw-dead-url-and-live-url">http://www.statistics.sk/mosmis/eng/run.html.{{dead link}} https://en.wikipedia.org/wiki/Main_Page.</ref>
```

Comment:
dead tags should really be attached to the end of the URL itself not, after the end of a sentence.